functions_intro/task2.js

```diff
@@ -0,0 +1,19 @@
+function isAdult(age) {
+  return Boolean(age >= 18);
```

Comment:
The greater than or equal (>=) operator returns true if the left operand is greater than or equal to the right operand, and false otherwise.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Greater_than_or_equal

Reply:
so no need to convert it to boolean explicitly

Reply:
Thanks! Removed Boolean conversion